examples/README.md

```diff
@@ -820,6 +821,32 @@ to the cluster using `kubectl` and will run on the specified node pool.
 
 [gke.yaml]: ../community/examples/gke.yaml
 
+### [ml-gke.yaml] ![community-badge] ![experimental-badge]
```

Comment:
Please make it bloody obvious that you absolutely must set `vars.authorized_cidr` or the module won't work as presently designed.

Reply:
I think as designed it is obvious as you are required to override at the command line.

But since our validation does not support leaving vars empty and I will have to populate with something, I agree it is dangerous that someone will miss this newance.  

I have added a warning at the top of the documentation. I have added additional comments in the blueprint. I also tested that terraform apply will fail if the user does not update the value.

Reply:
You might choose a strong word than "updated" and might even link to something like "whatismyip.com"? your decision before merging.

Reply:
Done.